PREINSTALL.md

```diff
@@ -12,37 +10,24 @@ When triggered, by adding a document to a specific Firestore collection, this ex
   to: 'RECIPIENT_OF_THE_MESSAGE',
 });
 ```
-
```

Comment:
I wouldn't remove new lines, they are actually should be there. There is md-lint plugin for VsCode that validates MD files content